OpenDreamClient/Interface/Controls/ControlBrowser.cs

```diff
@@ -113,7 +113,10 @@ public ControlBrowser(ControlDescriptor controlDescriptor, ControlWindow window)
             Stream stream;
             HttpStatusCode status;
             var path = new ResPath(newUri.AbsolutePath);
-            try {
+            if(!_dreamResource.EnsureCacheFile(newUri.AbsolutePath)) {
+                stream = Stream.Null;
+                status = HttpStatusCode.NotFound;
+            } else try {
```

Comment:
Put this `try` in a braced block to more clearly separate the catches from the if/else